Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -86,9 +88,12 @@ public async Task<SubmissionDetailsServiceModel> GetDetailsById(int submissionId
 
         var validationResult = this.submissionDetailsValidationService.GetValidationResult((submissionDetailsServiceModel, currentUser) !);
 
-        submissionDetailsServiceModel ??= new SubmissionDetailsServiceModel();
+        if (!validationResult.IsValid)
+        {
+            throw new BusinessServiceException(validationResult.Message);
+        }
 
-        submissionDetailsServiceModel.ValidationResult = validationResult;
+        submissionDetailsServiceModel ??= new SubmissionDetailsServiceModel();
```

Comment:
Validation of the submissionDetailsServiceModel for nullability should be handled in the validation service or before that. If we get to this point there should be a proper value for submissionDetailsServiceModel. Don't return an empty model because this is silencing what should be an error.

Reply:
Yes, it is actually being checked for nullability by the validation service, so this here is redundant and I removed it.